readme.md

```diff
@@ -20,17 +24,19 @@ ___
 - use `@font-face` for fonts
 - add attribute `data-qa="big"` for big search form, and `data-qa="small"` for small
 - add attribute `data-qa="keypress"` to input in big search form
+
 ---
+
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://SrTrace.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://SrTrace.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="942" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/3071d669-ae01-4104-b79f-19c0414925f4">